generator/vulkan/render.zig

```diff
@@ -426,6 +430,54 @@ fn Renderer(comptime WriterType: type) type {
             try self.renderWrappers();
         }
 
+        fn renderCommandEnums(self: *Self) !void {
+            try self.renderCommandEnumOfDispatchType(.base);
```

Comment:
So this function currently iterates over the registry six times...
Not sure if there's a better solution that doesn't significantly complicate things.

Reply:
A fix for this that would improve other parts as well is to classify commands by dispatch during the parsing phase.

Reply:
Lets leave it for now